apps/site/pages/docs/headless-cms-troubleshooting/error-permission-denied.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: 'Permission denied while running vtex cms sync'
+sidebar_label: 'Troubleshooting'
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+<header>
+# Permission denied error while running `vtex cms sync`
+</header>
+
+If you encounter a `Error: Permission denied` when using `vtex cms sync`, this issue is not related to the plugin version but rather to the user's permissions needed to execute the command and save resources.
```

Comment:
should we mention `vtex cms sync` or `faststore cms-sync`? 🤔

Reply:
Good question! This is an error happened with vtex cms sync. However, I could not reproduce the error using `faststore cms-sync`. Maybe the CMS team could help as well here

Reply:
This permission is always necessary for a successful CMS sync. Maybe it would be interesting to mention the two commands. cc: @vtex/content-management

Reply:
since here we are talking about faststore context in the starter, I think we can mention `faststore cms-sync`. @Mariana-Caetano

Reply:
another message that I received was:

> Error: User indicated by VtexIdclientAutCookie is not authorized to access the indicated resource.


maybe we can list both possible error messages 👍